### PR TITLE
Added log format support to prometheus spec

### PR DESCRIPTION
**Pull request by stijndehaes**
#### What this PR does / why we need it:

It adds support for setting the log format of prometheus so prometheus can log in json format.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus-operator/Chart.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 5.2.0
+version: 5.2.1
```
Comment (davidkarlsen) [inline stable/prometheus-operator/Chart.yaml]:
```suggestion
version: 5.3.0
```

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/prometheus-operator/Chart.yaml (hunk 1 above)
Comment (stijndehaes) [reply stable/prometheus-operator/Chart.yaml]: Done

**Pull request merged**